src/group_theory/group_action/defs.lean

```diff
@@ -95,27 +110,30 @@ class is_scalar_tower (M N α : Type*) [has_scalar M N] [has_scalar N α] [has_s
 is_scalar_tower.smul_assoc x y z
 
 section
-variables [monoid α] [mul_action α β]
+variables [monoid M] [mul_action M α]
 
-lemma smul_smul (a₁ a₂ : α) (b : β) : a₁ • a₂ • b = (a₁ * a₂) • b := (mul_smul _ _ _).symm
+@[to_additive] lemma smul_smul (a₁ a₂ : M) (b : α) : a₁ • a₂ • b = (a₁ * a₂) • b :=
+(mul_smul _ _ _).symm
 
-variable (α)
-@[simp] theorem one_smul (b : β) : (1 : α) • b = b := mul_action.one_smul _
+variable (M)
+@[simp, to_additive] theorem one_smul (b : α) : (1 : M) • b = b := mul_action.one_smul _
 
-variables {α}
+variables {M}
 
 /-- Pullback a multiplicative action along an injective map respecting `•`. -/
-protected def function.injective.mul_action [has_scalar α γ] (f : γ → β)
-  (hf : injective f) (smul : ∀ (c : α) x, f (c • x) = c • f x) :
-  mul_action α γ :=
+@[to_additive "Pullback an additive action along an injective map respecting `+ᵥ`."]
+protected def function.injective.mul_action [has_scalar M β] (f : β → α)
+  (hf : injective f) (smul : ∀ (c : M) x, f (c • x) = c • f x) :
+  mul_action M β :=
 { smul := (•),
   one_smul := λ x, hf $ (smul _ _).trans $ one_smul _ (f x),
   mul_smul := λ c₁ c₂ x, hf $ by simp only [smul, mul_smul] }
 
 /-- Pushforward a multiplicative action along a surjective map respecting `•`. -/
-protected def function.surjective.mul_action [has_scalar α γ] (f : β → γ) (hf : surjective f)
-  (smul : ∀ (c : α) x, f (c • x) = c • f x) :
-  mul_action α γ :=
+@[to_additive "Pushforward an additive action along a surjective map respecting `+ᵥ`."]
+protected def function.surjective.mul_action [has_scalar M β] (f : α → β) (hf : surjective f)
```

Comment:
Master has a definition below this one that also could be made additive; would you mind merging?

Reply:
What def are you talking about?